plugins/onepad/joystick.cpp

```diff
@@ -218,7 +218,7 @@ bool JoystickInfo::Init(int id)
 	// Keep the 4 hat buttons too (usb driver). (left pressure does not work with recent kernel). Moreover the pressure
 	// work sometime on half axis neg others time in fulll axis. So better keep them as button for the moment
 	u32 found_hack = devname.find(string("PLAYSTATION(R)3"));
-	if (found_hack != string::npos) {
+	if (found_hack != string::npos && (conf->options & PADOPTION_SIXAXIS_USB) == 0) {
```

Comment:
I was thinking box checked = ignore toggle events and use axes, box unchecked = process toggle events. It matches with the code in JoystickInfo::PollAxes that way.
```
		// Sixaxis, dualshock3 hack
		u32 found_hack = devname.find(string("PLAYSTATION(R)3"));
		if (found_hack != string::npos) {
			// The analog mode of the hat button is quite erratic. Values can be in half- axis
			// or full axis... So better keep them as button for the moment -- gregory
			if (i >= 8 && i <= 11 && (conf->options & PADOPTION_SIXAXIS_USB))
				continue;
			// Disable accelerometer
			if ((i >= 4 && i <= 6))
				continue;
		}
```